amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -173,7 +171,7 @@ export class SearchPage extends React.Component<SearchPageProps, SearchPageState
     ];
 
     return (
-      <div className="col-xs-12 col-md-offset-1 col-md-10">
+      <div>
```

Comment:
@ttannis originally removed this div but the test for `SearchPage › renderSearchResults › renders TabsComponent with correct props` failed. It looks like there is a difference between `content.find(TabsComponenet).props()` and `content.props()` where content is `<TabsComponent>`

Reply:
This is expected behavior according to Enzyme: https://github.com/airbnb/enzyme/issues/7

If I'm understanding that issue correctly, we might have to use `.props` to access the props directly instead of the method `.props()` which will return whatever information is associated with the root node of Enzyme's representation of this component?